### PR TITLE
support pgobject (JSONB)

### PR DESCRIPTION
**Pull request by macat**
Adding support for JSONB field. It is simply represented as a `stringValue`.

### CODE REVIEW

File: tests/test_models.py [hunk 1]
```diff
@@ -33,6 +40,15 @@ def __str__(self) -> str:
     uuid = 'e9e1df6b-c6d3-4a34-9227-c27056d596c6'
     assert Field.from_value(JavaUUID(uuid)) == Field(stringValue=uuid)
 
+    class PGObject:
```
Comment (koxudaxi) [inline tests/test_models.py]: @macat 
I think `PGObject` is a typo. Could you change it to `PGobject`?

Comment (macat) [reply tests/test_models.py]: Ups. sorry. fixed.

File: tests/test_models.py [hunk 2]
```diff
@@ -1,16 +1,23 @@
...
 
 
 def test_valid_field() -> None:
-    assert SqlParameter(name='abc', value=Field(stringValue='abc')).valid_value == 'abc'
-    assert SqlParameter(name='abc', value=Field(blobValue='abc')).valid_value == 'abc'
-    assert SqlParameter(name='abc', value=Field(doubleValue=0.1)).valid_value == 0.1
-    assert SqlParameter(name='abc', value=Field(isNull=True)).valid_value is None
-    assert SqlParameter(name='abc', value=Field(longValue=123)).valid_value == 123
-    assert SqlParameter(name='abc', value=Field(longValue=123)).valid_value == 123
+    assert SqlParameter(
```
Comment (koxudaxi) [inline tests/test_models.py]: @macat 
Sorry, I forgot to say about lint.
You can fix format with  `./scripts/format.sh`

Comment (macat) [reply tests/test_models.py]: sorry, had my autoformatter on.

**Comment by koxudaxi** [comment]
@macat 
I have released the version as `0.4.3`
Thank you very much.

**Pull request merged**